lib/exec.ex

```diff
@@ -204,4 +184,50 @@ defmodule QueryParser.Exec do
       end
     )
   end
+
+  defp exec_value_operator?("$eq", value, elem_value, elem, ctx) do
+    elem_value == exec_value(value, elem, ctx)
+  end
+
+  defp exec_value_operator?("$ne", value, elem_value, elem, ctx) do
+    elem_value != exec_value(value, elem, ctx)
+  end
+
+  defp exec_value_operator?("$lt", value, elem_value, elem, ctx) do
+    elem_value < exec_value(value, elem, ctx)
+  end
+
+  defp exec_value_operator?("$lte", value, elem_value, elem, ctx) do
+    elem_value <= exec_value(value, elem, ctx)
+  end
+
+  defp exec_value_operator?("$gt", value, elem_value, elem, ctx) do
+    elem_value > exec_value(value, elem, ctx)
+  end
+
+  defp exec_value_operator?("$gte", value, elem_value, elem, ctx) do
+    elem_value >= exec_value(value, elem, ctx)
+  end
+
+  defp exec_value_operator?("$exists", value, elem_value, elem, ctx) do
+    nil? = nil == elem_value
+    exec_value(value, elem, ctx) != nil?
+  end
+
+  defp exec_value_operator?("$size", value, elem_value, elem, ctx) do
+    if is_list(elem_value) do
+      elem_value |> length() == exec_value(value, elem, ctx)
+    else
+      false
+    end
+  end
```

Comment:
```suggestion
  defp exec_value_operator?("$size", value, elem_value, elem, ctx) when is_list(elem_value) do
      elem_value |> length() == exec_value(value, elem, ctx)
end
  
defp exec_value_operator?("$size", value, elem_value, elem, ctx) do
      false
end
```
Pattern matching is more efficient than if, it's a minor optimization but in this case, it doesn't cost much to do it.